qiling/os/struct.py

```diff
@@ -0,0 +1,244 @@
+from __future__ import annotations
```

Comment:
Is it possible to rename `struct` to some other names? It shadows the standard library `struct` and might cause some confusion for our users. (`ql.os` name clash had already caused a few confusion before)

Reply:
Couldn't come up with any reasonable alternative really.. any suggestions?